migrate.py

```diff
@@ -1652,6 +1652,9 @@ def main():
     spec = {}
 
     for spec_file in os.listdir(args.spec_dir):
+        if not spec_file.endswith('.yml'):
+            logger.debug('skipping %s as it is not a yaml file' % spec_file)
```

Comment:
"yaml" may confuse users with `*.yaml`
```suggestion
            logger.debug('skipping %s as it is not a `*.yml` file' % spec_file)
```